conf/messages.cy

```diff
@@ -113,8 +113,17 @@ generic.error.maxWhole=Rhaid i {0} gynnwys {1} digid cyfan ar y mwyaf
 generic.error.wholeNumber=Rhaid i {0} fod yn rhif cyfan
 generic.error.maxDecimals=Rhaid i {0} beidio â chynnwys mwy na {1} lle degol
 generic.error.invalid=Nid yw {0} yn ddilys
-generic.vrn.error.pattern=Rhowch rif TAW yn y fformat cywir
+generic.nino.error.pattern=CHANGE ME Enter {1} National Insurance number in the correct format
+generic.nino.error.required=CHANGE ME Enter {1} National Insurance number
+generic.nino.error.required= CHANGE ME Enter {1} National Insurance number
+generic.email.error.pattern= CHANGE ME  Enter {1} email address in the correct format, like name@example.com
+generic.utr.error.pattern=CHANGE ME Enter {1} Unique Taxpayer Reference in the correct format
+generic.utr.error.required=CHANGE ME Enter {1} Unique Taxpayer Reference
+generic.vrn.error.pattern=CHANGE ME Enter {1} VAT registration number in the correct format
+generic.vrn.error.required=CHANGE ME Enter {1} VAT registration number
 generic.vrn.error.digitcheck=Nid yw’r rhif TAW a nodoch yn bodoli. Nodwch rif TAW go iawn
+generic.paye.error.pattern=CHANGE ME Enter {1} PAYE reference in the correct format
+generic.paye.error.required=CHANGE ME Enter {1} PAYE reference
```

Comment:
Can we get rid of `CHANGE ME`? Or we can wait until Welsch translation arrives.

Reply:
may be wait before merging. Let's discuss on standup